meta-balena-honister/recipes-core/images/balena-image-flasher.bbappend

```diff
@@ -0,0 +1,4 @@
+# Make sure you have the raw resin image ready in DEPLOY_DIR_IMAGE (poky
+# rocko has the raw image in the deploy dir image after the do_image_complete
```

Comment:
this reference to rocko here is confusing too

Reply:
Thanks Alex, I addressed this in https://github.com/balena-os/meta-balena/pull/2368/commits/a1686f16702031576ac76dd08782418c67babaa7